howso/client/exceptions.py

```diff
@@ -133,35 +114,6 @@ def from_dict(cls, obj):
         return cls(message, code, status, url)
 
 
-class HowsoAuthenticationError(HowsoApiError):
```

Comment:
Keep the class, just drop the `from_openapi` method

Reply:
Addressed in c28e6de8dded39f04ba3206735fe885f931d8d2e